code/datums/Emote_system/emote.dm

```diff
@@ -428,13 +428,18 @@ VampyrBytes
 
 //Emote Cooldown System (it's so simple!)
 /datum/emote/proc/handle_emote_CD(var/mob/user)
-	if(user.emote_cd == 2) return 1			// Cooldown emotes were disabled by an admin, prevent use
-	if(user.emote_cd == 1) return 1		// Already on CD, prevent use
+	if(!user)
```

Comment:
Don't suppose you've still got the line number it runtimed on? The only way I can see for it to happen, this won't fix. It would be possible for `user` to be nullified during the spawn, and as the code after the spawn sets `emote_cd`, I would guess that's what actually happened.